chut/__init__.py

```diff
@@ -313,9 +315,15 @@ def command_line(self, shell=False):
                     break
             args.append(cmd)
         else:
+            import shlex
             args.extend(binary.split())
             for a in self.args:
-                args.extend(a.split())
+                if shell:
+                    args.append(a)
+                elif not isinstance(a, str):
```

Comment:
You'd rather use ```isinstance(a, (list, tuple))``` here. str is not the same thing in py2 and py3. That's always a mess